docs/plat/rpi3.rst

```diff
@@ -0,0 +1,297 @@
+ARM Trusted Firmware for Raspberry Pi 3
+=======================================
+
+.. section-numbering::
+    :suffix: .
+
+.. contents::
+
+The Raspberry Pi 3 is an inexpensive single-board computer that contains four
+Cortex-A53 cores, which makes it possible to have a port of the Arm Trusted
+Firmware.
```

Comment:
Would be good to have a reference to the official website (e.g. https://www.raspberrypi.org/products/raspberry-pi-3-model-b/).

Reply:
Done.